docs/guides/documentation-style.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from "@storybook/addon-docs";
+import showDo from "./assets/Atlantis_Docs_Show_Do.png";
+import showDont from "./assets/Atlantis_Docs_Show_Dont.png";
+
+<Meta title="Guides/Documentation styleguide" />
+
+# Documentation styleguide
+
+<br />
+
+## Purpose
+
+We should write clear, consistent docs to create a coherent experience for our
+consumers.
+
+<br />
+
+## Principles
+
+<br />
+
+### Minimum viable correctness
+
+| **✅ Do**                                                         | **❌ Don't**                                           |
+| :---------------------------------------------------------------- | :----------------------------------------------------- |
+| Provide the minimum amount of information necessary to be correct | Attempt to preempt every possible question or use case |
+
+Less is more. The more we write...
+
+- the more consumers must read
+- the more we must maintain
+
+We can always write more later as-needed to reflect questions that consumers
+bring to the team, or as the system evolves.
+
+<br />
+
+### Document for Jobber
+
+Follow Atlantis content guidelines for written content and component examples:
+
+- [formatting](../?path=/docs/content-formatting--docs)
+- [voice & tone](../?path=/docs/content-voice-tone--docs)
+- [vocabulary](../?path=/docs/content-product-vocabulary--docs)
+
+#### Relevance
```

Comment:
This is probably less critical, could drop only if we see this actually causing problems for people.

Why people do it:
- it's fun for the person doing it
- can help the author apply their mental model to a use case ("I know the different types of ships in Stargate, so I can use those types of ships as categories in this sectioned list")

Why I believe realistic data is more useful:
- the mentioned issue with readers not getting the reference (and the implied mental modal)
  - inconsistent consumer experience of different series/movies/books/etc being used as the reference point across the system depending on which dev wrote the example
- puts focus on why we're building the components/who we're building them for (if you have to look for realistic use cases, you gain a better understanding of how this element might be used in real life)

Reply:
Love this. I enjoy making things whimsical, but I've run into this situation at every company as they grow. Universal pop culture references are not universal at all. Not even the one you're thinking of right now in response to my statement.